python/ray/_private/accelerators/tpu.py

```diff
@@ -33,8 +33,10 @@
 NOSET_TPU_VISIBLE_CHIPS_ENV_VAR = "RAY_EXPERIMENTAL_NOSET_TPU_VISIBLE_CHIPS"
 
 # TPU VMs come with 4 chips per host and 2 tensorcores per chip.
+# TPU v5e is a special case which has 8 chips per host.
 # For more details: https://cloud.google.com/tpu/docs/system-architecture-tpu-vm
 TPU_NUM_CHIPS_PER_HOST = 4
```

Comment:
This is a bit ugly, I'm wondering if we should instead maintain a mapping between accelerator type and num chips?

Like:

```
TPU_NUM_CHIPS_PER_HOST_MAP = {
  "v5litepod-8": 8,
}
```

then below

```
num_visible_tpu_chips = len(visible_tpu_chips)
tpu_num_chips_per_host = TPU_NUM_CHIPS_PER_HOST_MAP.get(TPUAcceleratorManager.get_current_node_accelerator_type(), 4)
...
```

Reply:
I made it a static function, let me know if this works.